frontend/src/app/modules/api/client-config.js

```diff
@@ -1,5 +1,7 @@
 export const config = {
   baseURL: `${window.location.protocol}//${window.location.hostname}` +
-    `${process.env.REACT_APP_POST_URL_PORT}${process.env.REACT_APP_POST_URL_PATH_BASE}`,
+    `${process.env.REACT_APP_POST_URL_PORT !== "" 
```

Comment:
I guess, the whole point of having backticks is so that you don't use "+" for concatenation :D 
Also: 1. consider making an introductory variable 

```
const envPort: string = process.env.REACT_APP_POST_URL_PORT || '';
const colonPortOrNothing:string = envPort ? `:${envPort}`: ''
```
Or something similar :) 
Also, you are testing for a string equality against an empty string. This is quite fragile. What if the REACT_APP_POST_URL_PORT is not set, hence undefined?

Reply:
Great idea, done [here](https://github.com/salsita/foosball-rating/commit/eca8143fe7504de0175d2608f3b8d0f0f03bd9e7#diff-5696aa6f6ae27f14a99322ab211068ba)

I am using backticks mostly for their clear interpolation. The contatenation prevented whitespaces from breaking the URL. But I agree this way it looked rather ugly.